wheel/wheel.c

```diff
@@ -102,6 +105,8 @@ rt_err_t wheel_reset(wheel_t whl)
 }
 
 /** speed = rpm x 60 x 2 x PI x radius **/
+/** so : rpm = speed x 1.0f / (radius x 60 x 2 x PT) */
+/** then : rpm = speed * speed_to_rpm_transform --> precomputed */
 rt_err_t wheel_set_speed(wheel_t whl, double speed)
 {
     RT_ASSERT(whl != RT_NULL);

```

Comment:
The return value should be:
```
return wheel_set_rpm(whl, (rt_int16_t)(speed * whl->speed_to_rpm));
```

Reply:
Ooops, you're right this was caused by error during my rebase with master, I'll fix that.

Thank you :)

Reply:
> Ooops, you're right this was caused by error during my rebase with master, I'll fix that.
> 
> Thank you :)

By the way, my usual workflow is using ```git cherry-pick```. In this way, the workflow looks like:

1. Make changes
```
git clone https://github.com/your-user-name/rt-robot # Your forked repo
git checkout master
# Make changes
git add your-changed-files
git commit -m "Improved xxx"
git log --oneline (copy the commit hash)
```

2. New PR
```
git remote add upstream https://github.com/RT-Thread-packages/rt-robot
git fetch upstream master:upstream
git checkout upstream
git checkout -b upstream-my-modification
git cherry-pick  (copied commit hash)
git push origin master upstream-my-modification:upstream-my-modification
```

It is a good way to avoid conflicts and possible issues caused by ```git rebase```.